spec/models/card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Card do
+  it "original and translated texts cannot be equal" do
+    expect { Card.create!(original_text: "test",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.